<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>take_place</class>
 <widget class="QMainWindow" name="take_place">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>725</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>725</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Окно учета</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>706</width>
      <height>118</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>133</y>
      <width>80</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QLabel" name="use_count">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество использованных</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>230</y>
      <width>441</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="comment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Причина использования</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="count">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>260</y>
      <width>441</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="comment_line">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>140</y>
      <width>441</width>
      <height>81</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
